user_docs/en/changes.t2t

```diff
@@ -62,6 +62,7 @@ It is also updated with UIA enabled, when typing text and braille is tethered to
 - In Word, the landing cell will now be correctly reported when using the native Word commands for table navigation ``alt+home``, ``alt+end``, ``alt+pageUp`` and ``alt+pageDown``. (#15805, @CyrilleB79)
 - NVDA now resumes audio if the configuration of the output device changes or another application releases exclusive control of the device. (#15758, #15775, @jcsteh)
 - Contracted braille input works properly again. (#15773, @aaclause)
+- Braille is now updated when moving the navigator object between table cells in more situations (#15755)

```

Comment:
You may want to add your GitHub's name in the change log as a credit to recognize your work and in case people (e.g. devs) have questions on this topic.

Adding your GitHub in the change log is optional. Feel free to accept the suggestion or not:
```suggestion
- Braille is now updated when moving the navigator object between table cells in more situations (#15755, @Emil-18)
```